sdk/template/azure-template-stress/src/main/java/com/azure/sdk/template/stress/HttpGet.java

```diff
@@ -31,6 +34,7 @@ public class HttpGet extends ScenarioBase<StressOptions> {
     private static final ClientLogger LOGGER = new ClientLogger(HttpGet.class);
     private final HttpPipeline pipeline;
     private final URL url;
+    private final AtomicLong clientRequestId = new AtomicLong(Instant.now().getEpochSecond());
```

Comment:
I'm intrigued by this, we set the base value to the second the test started, and we keep incrementing by one each request. Does this better help correlate things than a slightly more verbose choice?

Reply:
added comment